lib/pages/view-post-page.js

```diff
@@ -68,7 +68,7 @@ export default class ViewPostPage extends BaseContainer {
 
 	commentEventuallyShown( comment ) {
 		const self = this;
-		const commentSelector = By.xpath( `//p[text() = "${comment}"]` );
+		const commentSelector = By.css( `p:contains("${comment}")` );
```

Comment:
Thanks! I could use some help; it's my first experience with React (just took a crash course over the weekend so I'm still new to it). I've got the code checked out and I'll be digging into it a bit tonight/tomorrow.

Also answers my other question on if I could do a pull request and work on it myself or not.